index.html

```diff
@@ -2494,34 +2522,90 @@
             <code>"web-nfc-only"</code>, and <var>messageId</var> is not
```

Comment:
It is initialized in the beginning of the steps, and it is updated when the Web NFC record is read. It is checked in the other steps.